README.md

```diff
@@ -1 +1,80 @@
-# Portfolio-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [Optional] I noticed that your readme file is for the hello micrverse project, kindly make this readme file personalized to the portfolio project. THis will make your work more professional.


- [Optional] Please in all future projects, kindly make sure that your branch name is the name of a feature or action that will be implemented in the branch. This is more professional. A good example of a branch name is `headline-section` and a bad example is `feature`.

- [Optional] Please in all future projects kindly make sure that the title of your Pull Request is a brief description of implemented changes. This is more professional.
Kindly check details of a [professional pull request here.]([url](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects))

Reply:
ok noted